splice_spark/src/test/scala/com/splicemachine/spark/splicemachine/TestContext.scala

```diff
@@ -169,9 +170,14 @@ trait TestContext extends BeforeAndAfterAll { self: Suite =>
   )
 
   override def beforeAll() {
-    sc = new SparkContext(conf)
-    SpliceSpark.setContext(sc)
-    spark = SparkSession.builder.config(conf).getOrCreate
+    spark = SpliceSpark.getSessionUnsafe
+    sqlContext = spark.sqlContext
+    spark.conf.set("spark.master", "local[*]")
+    spark.conf.set("spark.app.name", "test")
+    spark.conf.set("spark.ui.enabled", "false")
+    spark.conf.set("spark.app.id", appID)
```

Comment:
Does this make sense? We'd have to set those before instantiating the session, and this is already after that point, right?

Reply:
This isn't a SparkConf, as in initializeSparkSession().  This is a RuntimeConfig:
`  /**
   * Runtime configuration interface for Spark.
   *
   * This is the interface through which the user can get and set all Spark and Hadoop
   * configurations that are relevant to Spark SQL. When getting the value of a config,
   * this defaults to the value set in the underlying `SparkContext`, if any.
   *
   * @since 2.0.0
   */
  @transient lazy val conf: RuntimeConfig = new RuntimeConfig(sessionState.conf)`